data_collection/mobile/Piwik_PRO_SDK_for_Android.md

```diff
@@ -858,3 +858,58 @@ getTracker().setDryRunTarget(Collections.synchronizedList(new ArrayList<Packet>(
 ```
 * A dryRunTarget (required) – a data structure the data should be passed into ``List<Packet>`` type. Set it to null to disable dry run.
 
+
+## Cross platform tracking
+
+You can set up visitor recognition across both an app and a website. This allows you to recognize the same visitor when they move from a webview within the app to the website in their mobile browser.
+
+You can also track how users navigate from a website to an app using deep links.
+
+### Tracking users transitioning from the mobile browser to the mobile application
+
+Method ``setVisitorIdFromDeepLink`` sets a visitor ID from ``pk_vid`` parameter from provided deep link:
```

Comment:
```suggestion
Method `setVisitorIdFromDeepLink` sets a visitor ID from `pk_vid` parameter from provided deep link:
```
Double backticks are useful if you have backticks inside backticks https://www.markdownguide.org/basic-syntax/#escaping-backticks

Just FYI - it's ok for now.